release.sh

```diff
@@ -19,15 +18,12 @@ usage() {
         shift
     done
     echo "Usage:"
-    echo "./release.sh [OPTIONS] <VERSION_TO_RELEASE> [NEXT_VERSION]"
```

Comment:
Are we no longer bumping the version to `NEXT_VERSION-pre`?

Reply:
@kevinbarabash Yes. It was hard to determine which version to bump to. It's possible to bump the patch version, but it'd require two commits, release and bump, per a release. Another option is leave it `0.0.0-development` as packages with `semantic-release` do.

Reply:
I think it's fine not bumping.  Thanks for confirming.